### PR TITLE
Kepler mass redef

### PR DESCRIPTION
**Pull request by sblunt**
Medium-level API update. Under the new convention, `orbitize.system.System` objects are initialized either with a `mtot` parameter alone or with a `m0` and `m1` parameter. This allows for easier handling of dynamical mass fits with RV data + astrometry. Also, `orbitize.results.Results.save_results()` now saves the list of parameter labels in addition to the posterior, the loglikelihood, and the tau reference epoch.

### CODE REVIEW

**Comment by sblunt** [comment]
Addresses issue #122 and #123

File: orbitize/system.py [hunk 1]
```diff
@@ -121,28 +127,32 @@ def __init__(self, num_secondary_bodies, data_table, system_mass,
...
-        self.labels.append('mtot')
         if plx_err > 0:
             self.sys_priors.append(priors.GaussianPrior(plx, plx_err))
         else:
             self.sys_priors.append(plx)
         
         if self.fit_secondary_mass:
             for body in np.arange(num_secondary_bodies):
-                    self.sys_priors.append(priors.JeffreysPrior(1e-6, 1)) # in Solar masses for onw
+                self.sys_priors.append(priors.JeffreysPrior(1e-6, 1)) # in Solar masses for now
```
Comment (henry-ngo) [inline orbitize/system.py]: Didn't we rename this to LogUniform? Or would this be fixed when other changes are merged from master/other branches?

Comment (henry-ngo) [reply orbitize/system.py]: (maybe this is the conflict that @semaphoreP mentions!)

**Comment by henry-ngo** [comment]
Other than the above rename from JeffreysPiror->LogUniform, looks good :)

**Comment by sblunt** [comment]
Yup, that was it. Thanks @henry-ngo and @semaphoreP !

**Pull request merged**